### ISSUE TITLE
http://tsds.org/uri_templates Link

### ISSUE THREAD

**Issue by rweigel** [spase-group/spase-base-model#39]
This link is no longer active. I've asked Jeremy Faden to post a link here to the GitHub repo where this is stored.

**Comment by jbfaden** [comment]
I've been moving links to https://github.com/hapi-server/uri-templates/wiki/Specification, but I've been asked to move it to something like https://github.com/uri-templates/uri-templates/wiki/Specification.  (We don't control https://github.com/uri-templates, and I think we should continue to host it under https://github.com/hapi-server/uri-templates/ for now.)

**Comment by jbfaden** [comment]
I should ask if I can add the project:
https://github.com/spase-group/uri-templates

since this would be a logical place to house this project.

**Comment by rweigel** [comment]
Regarding https://github.com/spase-group/uri-templates, I think it should be https://github.com/uri-templates given that it is applicable outside of SPASE. (Same reason it does not belong under  https://github.com/hapi-server.)

**Comment by rweigel** [comment]
Link at https://spase-group.org/docs/conventions/index.html will also need to be updated.

**Comment by tressahelvey** [comment]
Chuck, can you take a look at this request and let me know if you can fix the issue? It would be ideal to talk this through with Zach too Thanks

**Comment by jbfaden** [comment]
Note the URI_Templates stuff was moved out of the hapi-server area to https://github.com/uri-templates-time/uri-templates/wiki/Specification

**Comment by Kurokio** [comment]
@rweigel Could you summarize what should be done to resolve this issue? What links need to be replaced, what should they be replaced with, and where are the currently broken ones found?

**Comment by rweigel** [comment]
They were found on the SPASE web page, at least.

For example, https://spase-group.org/docs/conventions/uri_templates_v1.pdf.

Seems like all the docs there should be in Zenodo too.

**Comment by Kurokio** [comment]
@rweigel Please verify on your end that I did not overlook any old links to the now replaced uri_templates.pdf.

**Comment by rweigel** [comment]
A search for "URI_Templates SPASE" gives

https://spase-group.org/docs/conventions/uri_templates_v1.pdf

Why not just link to the GitHub repo at https://spase-group.org/docs/conventions/index.html? Otherwise, your PDF will be out of date quickly. (Ideally, the page would post releases to Zenodo that you can link to, and I'll encourage them to do so.)

**Comment by Kurokio** [comment]
Good point @rweigel. I've updated it accordingly. Please review, thanks!

**Comment by rweigel** [comment]
Looks good.
________________________________
